lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
@jibsheet The flag in production is `ENABLE_GRADES_SAVED`, defined here: https://github.com/edx/edx-platform/blob/release/lms/envs/common.py#L364

We've since moved to a django admin config, but that change is only in master, not released. https://github.com/edx/edx-platform/pull/13329